_i18n/en.yml

```diff
@@ -156,15 +156,18 @@ security_page:
       us](site.baseurl/contact). You can also visit our [open-source repository](https://github.com/18F/identity-idp){:target="_blank"}.
 contact_page:
   content: |-
-    We’re experiencing extremely high traffic due to the Trusted Traveler Program launch today. Thank you for your patience as we work to improve our system performance.
+    ## Note for Trusted Traveler Program users:
+    To sign into the Trusted Traveler Program website, **you must create a new login.gov account.** login.gov does not have access to old GOES username, userID, or password and will not automatically recognize your email address. **Only your newly created login.gov username and password will allow you to sign in.**
 
-    **Note for Trusted Traveler Program users:** To sign into the Trusted Traveler Program website, you must create a new login.gov account. login.gov does not have access to old GOES username, userID, or password and will not automatically recognize your email address. Only your newly created login.gov username and password will allow you to sign in.
-
-    login.gov does not manage nor impact any of your existing membership, payments, appointments, or application status and is unable to respond to any questions about them.
+    login.gov does not manage nor impact any of your existing membership, payments, appointments, or application status and is unable to respond to any questions about them. If you have additional questions, **please refer to our [help page](site.baseurl/help)**.
 
     For questions about the Trusted Traveler Programs (Global Entry, GOES, Sentri, or Nexus), please contact CBP at [https://help.cbp.gov/app/ask](https://help.cbp.gov/app/ask){:target="_blank"}
 
-    For questions about creating an account or signing in with login.gov, look for answers to [common questions](site.baseurl/help). For other questions, please contact us [hello@login.gov](mailto:hello@login.gov). Please keep in mind that we’re experiencing a heavy volume of requests so it might take us several days to get back to you.
+    ## Find help using login.gov
+    First, check out answers to [common questions](site.baseurl/help). Still have a question or a problem? Call <span class="bold nowrap">844-875-6446</span> toll-free for more answers and to speak with an information specialist.
```

Comment:
Really not important today, given our current problems, but could we markup phone numbers using the tel protocol and then apply bold and nowrap to that? https://css-tricks.com/the-current-state-of-telephone-links/